guardrails/validator_base.py

```diff
@@ -448,10 +475,47 @@ def __init__(
             self.rail_alias in validators_registry
         ), f"Validator {self.__class__.__name__} is not registered. "
 
+    def chunking_function(self, chunk: str):
+        return split_sentence(chunk)
+
     def validate(self, value: Any, metadata: Dict[str, Any]) -> ValidationResult:
         """Validates a value and return a validation result."""
         raise NotImplementedError
 
+    def validate_stream(
+        self, chunk: Any, metadata: Dict[str, Any], **kwargs
+    ) -> ValidationResult:
```

Comment:
Is this PR the changeset where we decided to make this function a generator? Or is that coming later?

Reply:
I think that's coming later, or I would group that in with the new validation result schema